### PR TITLE
sorta working, selectedMeeting is wonky because of stack

### PR DESCRIPTION
**Pull request by Kayla-Mir**
added a toolbar to push content up from the two nav bars

### CODE REVIEW

File: src/components/BottomNavBar/BottomNavBar.jsx [hunk 1]
```diff
@@ -63,6 +63,7 @@ function BottomNavBar() {
   return (
     <div className="navBar">
       <ThemeProvider theme={theme}>
+        <Toolbar sx={{height: '9vh'}}/>
```
Comment (Kayla-Mir) [inline src/components/BottomNavBar/BottomNavBar.jsx:66]: added a hidden toolbar to force content out from behind the nav bar

File: src/components/MeetingHistory/MeetingHistory.jsx [hunk 2]
```diff
@@ -9,39 +10,45 @@ function MeetingHistory() {
...
-        style={{
-          backgroundColor: 'grey',
-          padding: '19px 0px',
-          marginBottom: '7px',
-          boxShadow: '0px 3px 5px rgba(0, 0, 0, .3)'
-        }}
-      >
-        <Typography
-          variant='h4'
+        <Box
```
Comment (Kayla-Mir) [inline src/components/MeetingHistory/MeetingHistory.jsx:13]: changed the div to a box, added some styling to work with box component and the toolbar

File: src/components/MeetingHistory/MeetingHistory.jsx [hunk 3]
```diff
@@ -9,39 +10,45 @@ function MeetingHistory() {
...
+              <MeetingHistoryItem key={item.id} item={item} />
+            </>
+          )
+        })
+        :
+        <Typography
+          sx={{ padding: '30vh 20vw', opacity: '60%' }}
         >
-          Meeting History
+          There are no meetings to display.
```
Comment (Kayla-Mir) [inline src/components/MeetingHistory/MeetingHistory.jsx:48]: things getting shoved down

File: src/components/MeetingHistoryItem/MeetingHistoryItem.jsx [hunk 4]
```diff
@@ -111,8 +111,8 @@ function MeetingHistoryItem({ item }) {
           </ExpandMore>
         </CardActions>
         <Collapse in={expanded} timeout="auto" unmountOnExit>
-          <CardContent sx={{ textAlign: 'center' }}>
-            <Typography>{item.summary}</Typography><br />
+          <CardContent sx={{ textAlign: 'center', padding: '0px' }}>
```
Comment (Kayla-Mir) [inline src/components/MeetingHistoryItem/MeetingHistoryItem.jsx:114]: removed the br tag and adjusted some margins/padding

File: src/components/MeetingNavBar/MeetingNavBar.jsx [hunk 5]
```diff
@@ -78,7 +78,7 @@ function MeetingNavBar({prop}) {
               </Grid>
               <Grid item xs={3}>
                 <IconButton
-                  onClick={() => history.push(`/meeting/details/${params.id}`)}
+                  onClick={() => history.push(`/meeting/${params.id}`)}
```
Comment (Kayla-Mir) [inline src/components/MeetingNavBar/MeetingNavBar.jsx:81]: updated to correct route, the rest of the file is just spacing

File: src/components/MeetingNavBar/MeetingNavBar.jsx [hunk 6]
```diff
@@ -180,6 +180,7 @@ function MeetingNavBar({prop}) {
             </Grid>
           </Box>
         </AppBar>
+        <Toolbar sx={{ height: '10vh' }}/>
```
Comment (Kayla-Mir) [inline src/components/MeetingNavBar/MeetingNavBar.jsx:183]: toolbar for shoving the content out of the nav bar

File: src/components/MeetingNotes/MeetingNotes.jsx [hunk 7]
```diff
@@ -50,7 +50,7 @@ function MeetingNotes() {
       }
     })
     toast.success(`${currentMeeting.meeting_title} updated!`)
-    history.push(`/meeting/details/${params.id}`);
+    history.push(`/meeting/${params.id}`);
```
Comment (Kayla-Mir) [inline src/components/MeetingNotes/MeetingNotes.jsx:53]: updated to the correct route

File: src/components/MeetingNotes/MeetingNotes.jsx [hunk 8]
```diff
@@ -64,54 +64,55 @@ function MeetingNotes() {
     <div>
       {/* have to give it a prop that matches the logic in the MeetingNavBar component */}
       <MeetingNavBar prop={'notes'} />
-      <div style={{ marginTop: '12vh' }}>
-        <Box
-          display="flex"
-          justifyContent="center"
-          alignItems="center"
```
Comment (Kayla-Mir) [inline src/components/MeetingNotes/MeetingNotes.jsx:71]: updated the div to a box like above

Comment (Kayla-Mir) [reply src/components/MeetingNotes/MeetingNotes.jsx:71]: the rest of the file is just stuff getting moved down

File: src/components/SelectedMeeting/SelectedMeeting.js [hunk 9]
```diff
@@ -41,7 +41,6 @@ const edit = (e) =>  {
     //map this out, research stack
     <div>
        <MeetingNavBar prop={'details'}/>
-     <div style={{marginTop: '86px'}}>
```
Comment (Kayla-Mir) [inline src/components/SelectedMeeting/SelectedMeeting.js:44]: div to box, set max width to the full screen (should probably be width but idk right now)

**Review by AllenJoeG — APPROVED** [review]
Thank you for great documentation in absentia

**Pull request merged**